gammapy/data/tests/test_data_store.py

```diff
@@ -88,6 +89,24 @@ def test_datastore_get_observations(data_store):
     observations = data_store.get_observations([11111, 23523], skip_missing=True)
     assert observations[0].obs_id == 23523
 
+    # Test that desired IRFs are loaded
+    hdu_table = data_store.hdu_table
```

Comment:
Please make this a separate test and remove the scope definition of the `data_store` fixture: then a new data store will are created and you can safely modify the hdu table.